easybuild/easyconfigs/l/libev/libev-4.33-GCC-11.2.0.eb

```diff
@@ -0,0 +1,37 @@
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'ConfigureMake'
+
+name = 'libev'
+version = '4.33'
+
+homepage = 'http://software.schmorp.de/pkg/libev.html'
+description = """A full-featured and high-performance (see benchmark) 
+event loop that is loosely modelled after libevent, but without its 
+limitations and bugs. It is used in GNU Virtual Private Ethernet, 
+rxvt-unicode, auditd, the Deliantra MORPG Server and Client, and many 
+other programs."""
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://dist.schmorp.de/libev']
+sources = ['%(name)s-%(version)s.tar.gz']
+
+checksums = [
+    '507eb7b8d1015fbec5b935f34ebed15bf346bed04a11ab82b8eee848c4205aea',  # libev-4.33.tar.gz
+]
+
+builddependencies = [
+    ('binutils', '2.37'),
+    ('Autotools', '20210726'),
```

Comment:
```suggestion
```

should only be needed if one is re-generating the configure file

Reply:
True, I removed that. Thanks.